docs/dg/dev/internationalization-and-multi-store/add-new-store-in-multi-db-setup.md

```diff
@@ -0,0 +1,170 @@
+* Verify that the new store’s database is correctly initialized and filled up with the demo data.
+
+#### Setting up additional deployment recipes
+
+It is convenient to create additional deployment install recipes (located under config/install folder) to setup a new and delete an existing stores, for testing purposes. Below is an example of such setup that proves to be working well on prcatice. We took the existing folder structure, and EU folder as a base, but you can introduce your structure:
+
+1. config/install/EU/setup-store.yml - contains everything needed to do a minimal setup of a new store(s):
+```
+env:
+  NEW_RELIC_ENABLED: 0
+command-timeout: 7200
+stores:
+  - { STORE-1 }
+  - { STORE-2 }
+  ...
+sections:
+  init-storage:
+    setup-search-create-sources:
+      command: "vendor/bin/console search:setup:sources -vvv --no-ansi"
+      stores: true
+  init-storages-per-store:
+    propel-migrate:
+      command: "vendor/bin/console propel:migrate -vvv --no-ansi"
+      stores: true
+  ...
+```
+2. config/install/EU/delete-store.yml - contains everything needed to remove an existing store(s):
+```
+env:
+    NEW_RELIC_ENABLED: 0
+command-timeout: 7200
+stores:
+  - { STORE-1 }
+  - { STORE-2 }
+  ...
+sections:
+    scheduler-clean:
+        scheduler-clean:
+            command: "vendor/bin/console scheduler:clean -vvv --no-ansi || true"
+            stores: true
+    clean-storage:
+        clean-storage:
+            command: "vendor/bin/console storage:delete -vvv --no-ansi"
+            stores: true
+    ...
+```
+In further sections below you’ll see how you can use your new custom recipe during the deployment in your main deployment yml file’s “SPRYKER_HOOK_DESTRUCTIVE_INSTALL“ parameter as following:
+
+```SPRYKER_HOOK_DESTRUCTIVE_INSTALL: "vendor/bin/install {STORES_GO_HERE} -r EU/setup-store --no-ansi -vvv"```
+
+or
+
+```SPRYKER_HOOK_DESTRUCTIVE_INSTALL: "vendor/bin/install {STORES_GO_HERE} -r EU/delete-store --no-ansi -vvv"```
+
+### Staging Setup
+#### Environment Configuration
+
+* Update the staging environment’s configuration to include the new store.
+* For database to be initialised, you will need to run a destructive deployment for your new store. To assure existing stores are not affected, you need to specify only new store code(s) in your deployment yml file (image.environment section), in `SPRYKER_HOOK_DESTRUCTIVE_INSTALL`. Example, for new PL and AT stores to be introduced:
+`SPRYKER_HOOK_DESTRUCTIVE_INSTALL: "vendor/bin/install PL,AT -r EU/destructive --no-ansi -vvv"`
+You can also use your custom recipe following the examples above (see “Setting up additional deployment recipes “)
+
+#### Support Requests
+* Open a support request to apply the new configuration to the staging environment. Attach all necessary files and provide detailed deployment instructions. You can also have the necessary configuration in a specific branch of your repository
```

Comment:
Let's list the files to attach to the request.
I'm also not sure which deployment instructions you mean, since Spryker own them, customers cannot change them.
```suggestion
* Open a support request to apply the new configuration to the environment. Attach deploy file and explain shortly expected changes, i.e. new DB should be created. In case you have the necessary configuration in a specific branch of your repository, provide a reference to it in the ticket, making sure support team has access to your code base.
```